core/src/main/antlr4/org/broadinstitute/dsde/rawls/expressions/parser/antlr/TerraExpression.g4

```diff
@@ -55,6 +55,6 @@ fragment INT: '0' | [1-9] [0-9]*;
 
 fragment EXP: [Ee] [+\-]? INT;
 
-SYMBOL: [a-zA-Z0-9_]+;
+SYMBOL: [a-zA-Z0-9_-]+;
```

Comment:
the removed output expression validation allowed dashes and also see
https://broadworkbench.atlassian.net/browse/CA-167